src/main/java/com/powsybl/network/conversion/server/NotificationService.java

```diff
@@ -49,24 +57,88 @@ private void sendCaseImportSucceededMessage(Message<String> message) {
         networkConversionPublisher.send("publishCaseImportSucceeded-out-0", message);
     }
 
+    private void sendNetworkExportStartMessage(Message<UUID> message) {
+        MESSAGE_OUTPUT_LOGGER.debug("Sending export network start message : {}", message);
+        networkConversionPublisher.send("publishNetworkExportStart-out-0", message);
+    }
+
+    private void sendNetworkExportSucceededMessage(Message<String> message) {
+        MESSAGE_OUTPUT_LOGGER.debug("Sending export network succeeded message : {}", message);
+        networkConversionPublisher.send("publishNetworkExportSucceeded-out-0", message);
+    }
+
+    private void sendCaseExportStartMessage(Message<UUID> message) {
+        MESSAGE_OUTPUT_LOGGER.debug("Sending export case start message : {}", message);
+        networkConversionPublisher.send("publishCaseExportStart-out-0", message);
+    }
+
+    private void sendCaseExportSucceededMessage(Message<String> message) {
+        MESSAGE_OUTPUT_LOGGER.debug("Sending export case succeeded message : {}", message);
+        networkConversionPublisher.send("publishCaseExportSucceeded-out-0", message);
+    }
+
     public void emitCaseImportStart(UUID caseUuid, String variantId, UUID reportUuid, String caseFormat, Map<String, Object> importParameters, String receiver) {
         sendCaseImportStartMessage(MessageBuilder.withPayload(caseUuid)
                 .setHeader(HEADER_VARIANT_ID, variantId)
                 .setHeader(HEADER_REPORT_UUID, reportUuid != null ? reportUuid.toString() : null)
                 .setHeader(HEADER_IMPORT_PARAMETERS, importParameters)
                 .setHeader(HEADER_RECEIVER, receiver)
-                .setHeader(HEADER_CASE_FORMAT, caseFormat)
+                .setHeader(HEADER_FORMAT, caseFormat)
                 .build());
     }
 
     public void emitCaseImportSucceeded(NetworkInfos networkInfos, String caseNameStr, String caseFormatStr, String receiver, Map<String, String> importParameters) {
         sendCaseImportSucceededMessage(MessageBuilder.withPayload("")
                 .setHeader(HEADER_NETWORK_ID, networkInfos.getNetworkId())
                 .setHeader(HEADER_NETWORK_UUID, networkInfos.getNetworkUuid().toString())
-                .setHeader(HEADER_CASE_FORMAT, caseFormatStr)
-                .setHeader(HEADER_CASE_NAME, caseNameStr)
+                .setHeader(HEADER_FORMAT, caseFormatStr)
+                .setHeader(HEADER_FILE_NAME, caseNameStr)
                 .setHeader(HEADER_RECEIVER, receiver)
                 .setHeader(HEADER_IMPORT_PARAMETERS, importParameters)
                 .build());
     }
+
+    public void emitNetworkExportSucceeded(UUID networkUuid, UUID studyUuid, UUID nodeUuid, UUID rootNetworkUuid, String userId, String exportUuid, String error) {
```

Comment:
This method is used for success and fail. Rename to emitNetworkExportDone or better I think: define 2 method